### DISCUSSION TITLE
Can't connect transducer sonar on BeaconStation_KeilaJoa

### DISCUSSION THREAD

**Discussion by Xardion495** [FakeFishGames/Barotrauma#14086] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Can't reach sonar transducer due the wall inside it

![20240603223739_1](https://github.com/FakeFishGames/Barotrauma/assets/17722887/58ff6916-482a-4b1f-b296-eface0254d58)
![20240603234413_1](https://github.com/FakeFishGames/Barotrauma/assets/17722887/1e1902d1-31e1-404e-b10c-31044552b344)

### Reproduction steps

1. Start game
2. Find unwired BeaconStation_KeilaJoa
3. Try to fix the transducer 

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Happens in both single player and multiplayer

### Version

v1.4.6.0 (Blood in the Water Update, hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by ubertpendragon** [comment]
That's every vanilla beacon, but connecting the transducer is not necessary (I wish) because none of the beacons have "use transducer" checked

**Reply by Xardion495** [reply]
Gosh, why it's designed like that? )
Everytime when i saw unwired god dammit transducer i wiring it. Only now I find out that the transducers are not working as normal person expecting
